test/controllers/licences.controller.test.js

```diff
@@ -188,4 +189,42 @@ describe('Licences controller', () => {
       }
     }
   })
+
+  describe('GET /licences/{id}/returns', () => {
+    beforeEach(async () => {
+      options = {
+        method: 'GET',
+        url: '/licences/7861814c-ca19-43f2-be11-3c612f0d744b/returns',
+        auth: {
+          strategy: 'session',
+          credentials: { scope: ['billing'] }
+        }
+      }
+    })
+
+    describe('when a request is valid and has returns', () => {
+      beforeEach(async () => {
+        Sinon.stub(ViewLicenceReturnsService, 'go').resolves(_viewLicenceReturns())
+      })
+
+      it('returns the page successfully', async () => {
+        const response = await server.inject(options)
+
+        expect(response.statusCode).to.equal(200)
+        expect(response.payload).to.contain('Returns')
+        //  Check the table titles
+        expect(response.payload).to.contain('Return reference and dates')
+        expect(response.payload).to.contain('Purpose and description')
+        expect(response.payload).to.contain('Due date')
+        expect(response.payload).to.contain('Status')
+      })
+    })
+
+    function _viewLicenceReturns () {
+      return {
+        activeTab: 'returns',
+        returns: [{}]
+      }
+    }
```

Comment:
Not saying it's right, but currently, for any helper functions like this we put them outside the `describe()` blocks at the bottom of the file.

Reply:
Done. Must have refactored at some point and missed this.